core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -430,6 +445,9 @@ func (ec *EthConfirmer) fetchAndSaveReceipts(ctx context.Context, attempts []Eth
 			return errors.Wrap(err, "saveFetchedReceipts failed")
 		}
 		promNumConfirmedTxs.WithLabelValues(ec.chainID.String()).Add(float64(len(receipts)))
+
+		duration := time.Now().Sub(attempts[i].EthTx.CreatedAt)
```

Comment:
Why not use the timestamp of the block number it was confirmed in?

Reply:
Doesn't look like we have that information without doing a query to an eth node, so for now going to use the `EthTx.CreatedAt` instead.